### PR TITLE
Cleaning up a few things

### PR DESCRIPTION
**Pull request by ryanfrancesconi**
updated AudioKit swxftlint yml to current syntax: unsure if desired but it didn't compile for me here.
syntax housekeeping, refactoring and safety on the audio unit manager and converter.

converted Dave's example RandomClips to use AKPlayer

### CODE REVIEW

File: Examples/macOS/RandomClips/RandomClips/ViewController.swift [hunk 1]
```diff
@@ -104,12 +129,12 @@ class ViewController: NSViewController {
 
                 drumPlayer.currentTime = 0
                 guitarPlayer.currentTime = 0
-
                 drumPlayer.prepare(withFrameCount: 44_100)
                 guitarPlayer.prepare(withFrameCount: 44_100)
 
-                drumLooper.schedule(from: 0, to: drumLooper.duration, avTime: nil)
-                guitarLooper.schedule(from: 0, to: guitarLooper.duration, avTime: nil)
+                drumLooper.play()
```
Comment (dave234) [inline Examples/macOS/RandomClips/RandomClips/ViewController.swift]: These will not be in sync if started this way.  Should be play(at: futureTime)

Comment (ryanfrancesconi) [reply Examples/macOS/RandomClips/RandomClips/ViewController.swift]: sorry typo... fixing

**Comment by ryanfrancesconi** [comment]
damnit

File: Examples/macOS/RandomClips/RandomClips/ViewController.swift (hunk 1 above)
Comment (dave234) [reply Examples/macOS/RandomClips/RandomClips/ViewController.swift]: Ok looks good as far as the example.  I didn't look at the rest.

**Pull request closed without merging**